src/app/component/modal/modal.tsx

```diff
@@ -0,0 +1,133 @@
+"use client";
+import React, { useState, useEffect } from "react";
+import Image from "next/image";
+import { useTranslations } from "next-intl";
+
+import { useConnect } from "wagmi";
+import { useAppDispatch, useAppSelector } from "@/redux/hooks";
+import WalletConnect from "@/app/svg/WalletConnect";
+import MetaMask from "@/app/svg/MetaMask";
+import CoinBase from "@/app/svg/CoinBase";
+import { toggleModal } from "@/redux/features/auth-slice";
+
+const Modal = ({}) => {
+  const [openModal, setOpenModal] = useState<boolean>(false);
+  const { connectors, connect } = useConnect();
+
+  const t = useTranslations("ConnectButton");
+  const dispatch = useAppDispatch();
+
+  // const { data: ensAvatar } = useEnsAvatar({ address })
+  // const { data: ensName } = useEnsName({ address });
+  // const {
+  //   connect,
+  //   connectors,
+  //   error,
+  //   isLoading,
+  //   pendingConnector,
+  //   isSuccess: connectIsSuccess,
+  //   data: connectData,
+  // } = useConnect();
+  // const { disconnect } = useDisconnect();
```

Comment:
Why is this commented?

Reply:
This code was not using anywhere that's way i just removed this